installation/docker.md

```diff
@@ -67,5 +67,9 @@ Alpine Linux uses Musl C library instead of Glibc. Musl is not fully compatible
 
 Our Docker containers images are deployed thousands of times per day, we take security and stability very seriously.
 
+<<<<<<< HEAD:installation/docker.md
```

Comment:
looks like a previous git conflict was merged as part of the commit

Reply:
Sorry- fixed